### DISCUSSION TITLE
Non-empty manifest read as empty

### DISCUSSION THREAD

**Discussion by ThePeach** [shakacode/shakapacker#515] [Q&A] [answered]
This is an odd one. On my local machine everything works as expected. I deploy the application on an ECS instance and when I ty to load the login page I get the error:
```
Processing by SessionsController#new as HTML
  Rendered sessions/new.slim (Duration: 0.6ms | Allocations: 384)
Completed 500 Internal Server Error in 5ms (ActiveRecord: 1.0ms | Elasticsearch: 0.0ms | Allocations: 1009)

ActionView::Template::Error (Shakapacker can't find admin.css in /app/public/packs/manifest.json.
[snip]

Your manifest contains:
{
}
...
[snip]
     8:     = stylesheet_pack_tag 'admin'
[snip]
app/views/sessions/new.slim:8
[snip]
```
when it's clear that the manifest in `public/packs/manifest.json` contains more than a few things:
```
# grep admin.css public/packs/manifest.json
  "admin.css": "/packs/css/admin.css",
  "css/admin.css.map": "/packs/css/admin.css.map",
          "/packs/css/admin.css"
```

I am at loss, where is it reading the empty manifest from??? 🤷

**Comment by Judahmeek** [comment, marked as answer]
Hi, @ThePeach. I'm sorry that we didn't respond to your query sooner. Github discussions don't notify the same way that Github issues do.

Were you able to resolve this issue?
